DOCS 5913 - Added Advanced Mule Concepts article. (#864)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/third-party-software-in-mule.adoc

endif::[]
:keywords: third, party, software, mule, 3rd

> Mule runtime engine includes third-party software as internal dependencies. Mule has dependencies on a subset of libraries from the included third-party software. Mule applications might also include additional third party dependencies managed through Maven, which are not included in the third-party software list shared below.

Reviewer: I wonder if "as internal dependencies" (found a few times in this doc) should be reworded to "upon which it is internally dependent"?

> Mule runtime engine includes third-party software as internal dependencies. Mule has dependencies on a subset of libraries from the included third-party software. Mule applications might also include additional third party dependencies managed through Maven, which are not included in the third-party software list shared below.

Reviewer: ```suggestion
Mule runtime engine is internally dependent on a subset of libraries from included third-party software. Mule applications might also include additional third-party dependencies (managed through Apache Maven) that are not included in the third-party software list.
```
